### PR TITLE
Using enmerkar latest ver instead of commit hash

### PR DESCRIPTION
**Pull request by awais786**
Using enmerkar latest ver instead of commit hash
Rename the repo name.
update setup information.

https://openedx.atlassian.net/browse/BOM-1637

### CODE REVIEW

**Comment by awais786** [comment]
@jmbowman can you please create a new fork I dont have permission.

File: .travis.yml [hunk 1]
```diff
@@ -23,3 +23,14 @@ install:
 
 script:
   - tox -e $TOX_ENV
+
+deploy:
+  provider: pypi
+  user: edx
+  password:
+    secure: S3YrdGTgH3AXDN5EzMVLqyUvp/RvPc8lVo3Z5KrU3P9E87cjWyBUI+jVUk38lBUkO06n9ACbWt44f7H6n2GQeKCf6ZUZa69QgxXPRQZP+XiEGwtlv2YgatqeqNG7y6WTp3261mJ5kXuEcKBak7uNpru3gazEUxgc1EtbrsNPVS3oMChyIsi3j3KJuAq+h0AHibx1XMgWToU9aCbmrznjs18dXPmu+VhXyqJGliz79zxn5mvSZjCFOYwZtie/uD/g7+4V5wk75aDAfr4GYTbvvIvuDxyopZsaSRUB89aNWlpKKbOlsL5DuJWSkcY2I31tAQ3z3FtIJPKBFC44N81jwk+JdD4APq1qzeu3DMO+O8xJbUi8VZ6QgWiU17WnIFff0s9sTtmBdDRZAfx5XR4kqMLiVADSeR2XNfivP7/57LgEog0p0zrS2VnSbvvvk1lxfLxxwr3qk0DubP5SqgtqCrBfj/C5/iz5gg24c56kzfvf0BWF1vleez2Se/zQlICODpaT+Jf/2kPKFaaM3vwAKnCR/wD+Q9p6B1Dq8eb2ZRs4ucEkAuJtGfaKZrNUInt6h1bF7StnjyaXZnZWRbBRLMudoFEVOLg5pbEZiktYNuZWleYU120YUoz6O1mZOjt/FzPgXOu5o1tHN8OdcDWaEIzAO78LrIKAf6vqMtTavDM=
```
Comment (awais786) [inline .travis.yml:31]: I used this `secure` from django splash.

File: .travis.yml [hunk 2]
```diff
@@ -23,3 +23,14 @@ install:
 
 script:
   - tox -e $TOX_ENV
+
+deploy:
+  provider: pypi
+  user: edx
+  password:
+    secure:
```
Comment (awais786) [inline .travis.yml:31]: working on it.

**Comment by awais786** [comment]
@jmbowman Please review this PR. I will merge it and in other PR finish the travis pypi task

File: setup.py [hunk 3]
```diff
@@ -58,25 +58,25 @@ def is_requirement(line):
...
 setup(
-    name='django-babel-underscore',
+    name='enmerkar-underscore',
     version=version,
     description='Implements a underscore extractor for django-babel.',
     long_description=read('README.rst') + u'\n\n' + read('HISTORY.rst'),
     author='Christopher Grebs',
     author_email='cg@webshox.org',
-    url='https://github.com/EnTeQuAk/django-babel-underscore',
+    url='https://github.com/Zegocover/enmerkar',
```
Comment (jmbowman) [inline setup.py:67]: This should be the URL of this repo.  And the author and author_email should also be updated to the standard edX values.

File: setup.py [hunk 4]
```diff
@@ -58,25 +58,25 @@ def is_requirement(line):
...
     [babel.extractors]
-    underscore = django_babel_underscore:extract
+    underscore = enmerkar_underscore:extract
     """,
     zip_safe=False,
     license='BSD',
-    keywords='django-babel-underscore',
+    keywords='enmerkar-underscore',
     classifiers=[
         'Development Status :: 2 - Pre-Alpha',
```
Comment (jmbowman) [inline setup.py:81]: Maybe update the development status while we're at it?  We've been using this in production for years, so pre-alpha doesn't seem appropriate.

**Pull request merged**